### PR TITLE
Adds new partner

### PR DESCRIPTION
**Pull request by oriserra**
Signed-off-by: oriserra <o.serra@sease.io>

### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: sease.markdown [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+---
+name: Sease
+# upload your logo to the following directory - must be square
+logo: '[/assets/media/partners/sease.png](https://sease.io/wp-content/uploads/2022/09/SEASE-FAVICON-gradient-24-4.png)'
```
Comment (krisfreedain) [inline sease.markdown:5]: Please upload your logo as part of this pull request, not hyperlinked over to an external location.

File: sease.markdown [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+link: sease.io
```
Comment (krisfreedain) [inline sease.markdown:6]: Please fully qualify your url - https://sease.io/

**Review by krisfreedain — CHANGES_REQUESTED** [review]
hello @oriserra - glad to have you coming on as a partner! Two small changes needed. thanks

File: sease.markdown (hunk 1 above)
Comment (oriserra) [reply sease.markdown]: Hello, we have done this but it is still giving error. Could you please help me fix it? Thank you!

**Comment by krisfreedain** [comment]
hello @oriserra -- I'll work on a resolution to this and get you added to the partner page. Will update here as soon as resolved

**Comment by krisfreedain** [comment]
Hello @oriserra - I've resolved the issues with DCO in https://github.com/opensearch-project/project-website/pull/1116 -- welcome to the [partners page](https://opensearch.org/partners)! Will close this one out.

**Pull request closed without merging**